### PR TITLE
LPS-201893 Use new taglib to the Fragment

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
- LPS-201893 use the new tag on the fragment
- LPS-201893 remove the .lfrbuild-releng-ignore to let the saved content tag be used by other modules without failure.

### CODE REVIEW

File: modules/apps/fragment/fragment-impl/src/main/java/com/liferay/fragment/internal/renderer/SaveContentFragmentRenderer.java [hunk 1]
```diff
@@ -63,8 +70,43 @@ public void render(
...
+		SavedContentTag savedContentTag = new SavedContentTag();
+
+		savedContentTag.isViewMode(fragmentRendererContext.isViewMode());
+
+		Tuple displayObject = getDisplayObject(
+			fragmentRendererContext, httpServletRequest);
+
+		savedContentTag.setClassName(
+			GetterUtil.getString(displayObject.getObject(0)));
+
```
Comment (adolfopa) [inline modules/apps/fragment/fragment-impl/src/main/java/com/liferay/fragment/internal/renderer/SaveContentFragmentRenderer.java:87]: Optional: I think this newline shouldn't be here; both set* methods take its input from the same place.

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:test:relevant

File: modules/apps/fragment/fragment-impl/src/main/java/com/liferay/fragment/internal/renderer/SaveContentFragmentRenderer.java (hunk 1 above)
Comment (AliciaGarciaGarcia) [reply modules/apps/fragment/fragment-impl/src/main/java/com/liferay/fragment/internal/renderer/SaveContentFragmentRenderer.java:87]: thanks, I will add an commit removing this newline

**Comment by AliciaGarciaGarcia** [comment]
ci:stop

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Comment by SylviaLuan** [comment]
ci:forward:force

**Pull request closed without merging**